test/api/v3/integration/hall/GET-hall_heroes_heroId.test.js

```diff
@@ -67,4 +67,16 @@ describe('GET /heroes/:heroId', () => {
     expect(heroRes.auth.local).not.to.have.keys(['salt', 'hashed_password']);
     expect(heroRes.profile).to.have.all.keys(['name']);
   });
+
+  it('returns hero data given username without case sensitivity', async () => {
```

Comment:
hey @randi2kewl thanks for the extra test but I'm not sure what's being tested here. I think a more useful test would be to make a search request using the wrong casing for the username and check that the correct user is returned anyway. so for example search for `TestuPPerCaseName123` and check that it works

Reply:
Adjusted. Hopefully this is a better suited test.